<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GS4PS2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="ConnectionLayout">
            <property name="title">
             <string>Connection</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QGridLayout" name="ConnectionLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item row="1" column="1">
                <widget class="QLineEdit" name="txtHost">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>434</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="lblState">
                 <property name="text">
                  <string>Not Connected</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Connection Status</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>IP address</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Version</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="lblVersion">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btnConnect">
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDisconnect">
               <property name="text">
                <string>Disconnect</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkUDPTTYConsole">
            <property name="text">
             <string>Toggle Console</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkReplay">
            <property name="text">
             <string>Replay on dump end</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grpDumpData">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>State Size</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Size</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>State Version</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Type:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_15">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>CRC</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="lblDumpStateVersion">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="lblDumpSize">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="lblDumpCRC">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="lblDumpStateSize">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="lblDumpType">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtDumpPath"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btnBrowse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Path</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listDumpFiles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ClickableLabel" name="lblDumpPreview">
            <property name="maximumSize">
             <size>
              <width>215</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFrames">
       <attribute name="title">
        <string>Frames</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Circuit 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listC1Frames"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Circuit 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listC2Frames"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Preview latest</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QRadioButton" name="btnPreviewC1">
               <property name="text">
                <string>Circuit 1</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="btnPreviewC2">
               <property name="text">
                <string>Circuit 2</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ClickableLabel" name="lblCurrentFrameImg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>Next Frame</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStop">
    <property name="title">
     <string>Stop</string>
    </property>
   </widget>
   <addaction name="menuItem"/>
   <addaction name="menuStop"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkUDPTTYConsole</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>chkUDPTTY_Changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>btnConnect_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDisconnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>btnDisconnect_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtDumpPath</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>txtDumpPath_TextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnBrowse_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listDumpFiles</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listDumpFiles_itemClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listDumpFiles</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listDumpFiles_itemDoubleClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listC1Frames</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listFrame_currentItemChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listC2Frames</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listFrame_currentItemChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listC1Frames</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listFrame_currentItemChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listC2Frames</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listFrame_currentItemChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lblDumpPreview</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>screenshotWidget_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chkUDPTTY_Changed(int)</slot>
  <slot>btnConnect_Clicked()</slot>
  <slot>btnDisconnect_Clicked()</slot>
  <slot>btnBrowse_Clicked()</slot>
  <slot>txtDumpPath_TextChanged(QString)</slot>
  <slot>listDumpFiles_itemClicked(QListWidgetItem*)</slot>
  <slot>listDumpFiles_itemDoubleClicked(QListWidgetItem*)</slot>
  <slot>screenshotWidget_Clicked()</slot>
  <slot>listFrame_currentItemChanged(QListWidgetItem*)</slot>
 </slots>
</ui>
